packaging/windows/WIXInstallers/KeybaseApps/KeybaseApps.wxs

```diff
@@ -260,4 +261,12 @@
               Execute="immediate"
               Return="ignore"/>
   </Fragment>
+  <Fragment>
+    <CustomAction Id="LogoutUser"
+              Directory="INSTALLFOLDER"
+              ExeCommand="[INSTALLFOLDER]keybaserq.exe -wait &quot;[INSTALLFOLDER]keybase.exe&quot; logout"
```

Comment:
logout automatically does the deleting/shredding of the right files, right?

Reply:
Yes, see secret_store_file.go
Tested an installer built from this branch: http://s3.amazonaws.com/prerelease.keybase.io/windows/Keybase_1.0.45-20180326142420%2B531e573.386.exe